README.md

```diff
@@ -1 +1,192 @@
-# metrics-app
\ No newline at end of file
+<a name="Metrics WebApp"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
```

Comment:
- [OPTIONAL] Kindly remove the Microverse logo. If you have your logo, you can replace the Microverse logo with it, but if you don't, kindly remove it. 👍